dsl-reference.md

```diff
@@ -66,7 +66,7 @@ A [workflow](#workflow) serves as a blueprint outlining the series of [tasks](#t
 | document | [`document`](#document) | `yes` | Documents the defined workflow. |
```

Comment:
Side note, there is a little bug from @fjtirado's PR, if you can fix it at the same time. In the TOC:
`[Export] (#export)`
-> `[Export](#export)`